smsVoIncident.js

```diff
@@ -0,0 +1,40 @@
+const qs = require('qs');
+const got = require('got');
+const _ = require('lodash');
+
+
+exports.handler = function(context, event, callback) {
+    const { ROUTING_KEY, VICTOROPS_TWILIO_SERVICE_API_KEY } = context;
+    console.log(`${ROUTING_KEY} ${VICTOROPS_TWILIO_SERVICE_API_KEY}`);
+
+    var got = require('got');
+
+    let twiml = new Twilio.twiml.MessagingResponse();
+    twiml.message({ to: event.From }, 'Incident Created');
+
+    var alert = {
+        monitoring_tool: 'Twilio',
+        message_type: 'critical',
+        entity_display_name: `${event.Body}`,
+        state_message: `From ${event.From} -- ${event.Body}`,
+        entity_id: `${event.From}`
```

Comment:
Is there something available to us that would be more unique than `event.From`?  Does the sms come with a CallSid or anything like that?

Reply:
i decided on even.From so that repeat text messages from a single number could not spam incidents and that subsequent messages would wrap up into one existing incident. Open to other opinions though.

Reply:
That's reasonable.   More of a product question, so maybe just bring that up.